subcontracted_service/models/warehouse.py

```diff
@@ -22,16 +22,7 @@ def _get_buy_route(self):
     def _get_vals_for_proc_rule_subcontracting(self):
         self.ensure_one()
         picking_type = self.in_type_id
-        if not picking_type:
```

Comment:
Please, could you explain why there is no need to validate picking_type? 🤔

Reply:
It was added due to this issue: https://github.com/OCA/purchase-workflow/issues/551